ssm.py

```diff
@@ -0,0 +1,11 @@
+import boto3
+
+
+def getPasswords():
+  ssm = boto3.client('ssm')
+  passwrd = ssm.get_parameter(Name='/prod/nyp/snowflake-db/stack-user-pasword', WithDecryption=True)
```

Comment:
Do not hardcode the deployment. Read it from the environment so that the `default` database is set automatically (https://analyticsmd.atlassian.net/browse/DP-1372)

Reply:
Due to security breach we cannot have all deployments passed at once. We will have to segregate based on containers running for that deployment.

Reply:
`nyp` is hard coded. Get the deployment from an environment variable